pkgs/tools/networking/udpreplay/default.nix

```diff
@@ -0,0 +1,22 @@
+{ stdenv, cmake, libpcap, fetchgit, lib }:
+stdenv.mkDerivation rec {
+  name = "updreplay";
+  version = "1.0.0";
+  buildInputs = [ cmake libpcap ];
+  src = fetchgit {
```

Comment:
Please use fetchFromGitHub instead of fetchgit

Reply:
Initially I couldn't get `fetchFromGitHub` working (I initially specified a commit hash instead of the v1.0.0 tag) and thus opted for fetchgit. However, trying again now I was able to use `fetchFromGitHub` just fine.